extern/rust/Makefile.mk

```diff
@@ -6,13 +6,13 @@ LIBCORE_DIR=$(BUILD_DIR)/core-$(RUSTC_VERSION)
 
 $(EXTERN_DIR)rust/rustc-$(RUSTC_VERSION)-src.tar.gz:
 	@echo "Need libcore to compile Tock: fetching source $(@F)"
-	@wget -q -O $@ https://github.com/rust-lang/rust/archive/$(RUSTC_VERSION).tar.gz
+	@wget -q -O $@ https://github.com/rust-lang/rust/archive/$(RUSTC_VERSION).tar.gz --no-check-certificate
```

Comment:
It's a problem of wget. As I found out, version 1.14 should not have this problem (related to SNI support).

What version do you guys use? I downloaded one for windows (http://gnuwin32.sourceforge.net/packages/wget.htm) which is 1.11.4 and this one has a problem. I'll get the newer one 1.14 and above, which should fix this. I'll retest and let you know.